src/libxrpl/protocol/Permissions.cpp

```diff
@@ -117,20 +132,45 @@ Permission::getGranularTxType(GranularPermissionType const& gpType) const
     return std::nullopt;
 }
 
-bool
-Permission::isDelegatable(std::uint32_t const& permissionValue) const
+TER
+Permission::isDelegatable(
+    std::uint32_t const& permissionValue,
+    Rules const& rules) const
 {
     auto const granularPermission =
         getGranularName(static_cast<GranularPermissionType>(permissionValue));
     if (granularPermission)
         // granular permissions are always allowed to be delegated
-        return true;
+        return tesSUCCESS;
 
     auto const it = delegatableTx_.find(permissionValue - 1);
```

Comment:
this is not related to the change, but should  probably use `permissionToTxType` function